wdl-analysis/src/config.rs

```diff
@@ -108,6 +114,26 @@ impl Config {
             inner: Arc::new(inner),
         }
     }
+
+    /// Return a new configuration with the previous ignore filename replaced by
+    /// the argument.
+    ///
+    /// Specifying `None` for `filename` disables ignore behavior. This is also
+    /// the default.
+    ///
+    /// `Some(filename)` will use `filename` as the ignorefile basename to
+    /// search for. Child directories _and_ parent directories are searched
+    /// for a file with the same basename as `filename` and if a match is
+    /// found it will attempt to be parsed as an ignorefile. Errors during
+    /// this parsing are discarded, as it is a non-critical feature and should
+    /// not block further anlaysis.
```

Comment:
should this doc comment link out to something that defines ignorefiles, or is it just expected anyone this deep in the guts of `wdl_analysis` is familiar with them?